docs/pages/includes/database-access/aws-create-db-configuration.mdx

```diff
@@ -0,0 +1,54 @@
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Create a configuration file at `/etc/teleport.yaml` (or modify an existing
+one):
+
+```code
+$ sudo cat > /etc/teleport.yaml <<EOF
+version: v3
+teleport:
+  nodename: "database-agent"
+  data_dir: "/var/lib/teleport"
+  proxy_server: <Var name="mytenant.teleport.sh:443" />
+  auth_token: "/tmp/token"
+proxy_service:
+  enabled: "no"
+auth_service:
+  enabled: "no"
+ssh_service:
+  enabled: "no"
+db_service:
+  enabled: "yes"
+  # Monitor dynamic resources from Auto-Discovery, tctl, Terraform:
+  resources:
+  - labels:
+      env: prod
+    aws:
+      # Optional AWS role that the Database Service will assume to access
+      # this database.
+      assume_role_arn: ""
+      # Optional AWS external ID that the Database Service will use to assume
+      # a role in an external AWS account.
+      external_id: ""
+  # Or, define an AWS {{ type }} database statically:
+  databases:
+  - name: example-{{ matcher }}
+    protocol: {{ protocol }}
+    uri: <Var name="{{ example }}" />
+    aws:
+      # Optional AWS role that the Database Service will assume to access
+      # this database.
+      assume_role_arn: ""
+      # Optional AWS external ID that the Database Service will use to assume
+      # a role in an external AWS account.
+      external_id: ""
+    static_labels:
+      env: prod
+EOF
+```
```

Comment:
Same remark as  in docs/pages/includes/database-access/aws-create-db-configuration-with-account-id.mdx